### DISCUSSION TITLE
Moving Box 1 & 2

### DISCUSSION THREAD

**Discussion by Aariq** [SORTEE-Github-Hackathon/manuscript#49] [General]
So I added two "boxes" in PR #43, but they don't show up in the built paper.  I realize now how the paper gets built from the .md docs.  Should I just include the content of my .md docs in the introduction section?   I don't know exactly how to do this for the .md that's build from a .rmd file.  Is there a simple markdown way of including something like child documents?  I can also just manually add the markdown table, but it's really hard to read, IMO.

**Comment by Aariq** [comment]
`content/box1.definitions.md` is easy to just copy/paste into the `02.introduction.md` file,
but `content/box2.profiles.Rmd` which gets knit to `content/box2.profiles.md` is not so easy to incorporate.

**Comment by Aariq** [comment]
Maybe for now I can output the contributor profiles table as an image, and include it in the introduction like a figure?

**Comment by Aariq** [comment]
@kaitlyngaynor, I saw you were interested in helping with this table.  Right now the text of the table is here if you'd like to edit: https://docs.google.com/spreadsheets/d/1R0_NGhAWJ3cxGKQVh06eHKxHGMCN6x5aNL3_qR3nAI0/edit#gid=0

**Comment by kaitlyngaynor** [comment]
Thank you. I have requested editing permissions (it was just view only for me).

To clarify, the "roles" are somewhat akin to RStudio "[learner personas](https://rstudio-education.github.io/learner-personas/)," right? We'll want to clarify in a caption that these are just some examples of GitHub user types rather than prescriptive roles that people should fill.

And that's in contrast to the second table, which (as I understand it) are actual terms from GitHub.

Could we pull into two separate tabs in the Google Sheet for easier editing?

**Reply by Aariq** [reply]
> To clarify, the "roles" are somewhat akin to RStudio "[learner personas](https://rstudio-education.github.io/learner-personas/)," right? We'll want to clarify in a caption that these are just some examples of GitHub user types rather than prescriptive roles that people should fill.

Yeah, exactly.  My hopes were to give people some ideas of how GitHub could be useful even in situations where not all of your collaborators are coders willing to learn version control.  My inspiration was to point out benefits to using GitHub even if, for example, you are a grad student and your advisor has no time to learn git.  Maybe you can at least use GitHub to share your results via GitHub pages, or maybe you can convince them to make comments on your code via Issues, etc.

>And that's in contrast to the second table, which (as I understand it) are actual terms from GitHub.

Yes, the other box is just definitions of git/GitHub jargon that I've moved into the introduction as a sub-section called Box 1 (for now at least). 

>Could we pull into two separate tabs in the Google Sheet for easier editing?

Right now the code just pulls the first sheet to make a table.  I think the GitHub definitions is easy enough to edit just as markdown.
